go/kbfs/libkbfs/favorites.go

```diff
@@ -631,10 +675,26 @@ func (f *Favorites) RefreshCache(ctx context.Context) {
 		ctx:     context.Background(),
 	}
 	f.wg.Add(1)
+
+	if async {
+		favResult, err := f.config.KBPKI().FavoriteList(ctx)
```

Comment:
This seems like the opposite of async -- it's synchronously blocking on a call to the service, right? Did you mean `!async` above?